sympy/assumptions/tests/test_smtask.py

```diff
@@ -0,0 +1 @@
+from sympy.assumptions.smtask import z3ask
from sympy.core.symbol import symbols
from sympy.testing.pytest import raises, XFAIL
from sympy.assumptions.ask import Q
x, y, z = symbols('x y z')
def test_z3ask():
    assert z3ask(y > 0, (y < -x**2 +.5)) is None
    assert z3ask(y > 0, (y < -x**2 +.5) & Q.integer(y)) is False
    assert z3ask(x > z, (x > y) & (y > z)) is True


@XFAIL
def test_z3ask_fail():
    assert z3ask((y > 0), (y > 2) & Q.imaginary(y)) is False # currently returns True
    raises(ValueError, lambda:  z3ask((y > 0), (y > 2) & Q.imaginary(y)))
    assert z3ask((y > 0) & Q.imaginary(y), (y > 2))  is False # currently returns True
    assert z3ask(Q.real(y), y>2) is False
    assert z3ask( x > y, Q.infinite(x) & Q.positive(y) & Q.positive(x)) is True

```

Comment:
It looks like you may not have git configured well for line endings. See https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings